crates/holochain/src/conductor/conductor.rs

```diff
@@ -921,6 +921,11 @@ where
         Ok(active_apps.keys().cloned().collect())
     }
 
+    pub(super) async fn list_inactive_apps(&self) -> ConductorResult<Vec<InstalledAppId>> {
```

Comment:
I think we really need to get the [`DeactivationReason`](https://github.com/holochain/holochain/blob/7c81758a4644d4156aa7a56ab661d007e0a0f701/crates/holochain_types/src/app.rs#L385) returned here also, not just the apps. so the result would be a pair.  We need to know if it can be activated again and the reason will tell us.